int8 cursor
---
int8 response
string transcription
